Examples/Movies/android/app/src/main/java/com/facebook/react/movies/MoviesActivity.java

```diff
@@ -35,7 +36,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     mReactInstanceManager = ReactInstanceManager.builder()
         .setApplication(getApplication())
-        .setBundleAssetName("MoviesApp.android.bundle")
+        .setJSBundleLoader(JSBundleLoader.createAssetLoader("MoviesApp.android.bundle"))
```

Comment:
I mean, you could probably argue that all that is an implementation detail and we should just do some major renaming such that ReactInstanceManager becomes ReactBridge :P